imap_processing/tests/ultra/unit/test_ultra_l1b_annotated.py

```diff
@@ -9,7 +9,7 @@
 
 
 @pytest.fixture()
-def kernels(spice_test_data_path):
+def kernels(spice_test_data_path, _download_de440s):
```

Comment:
The `_download_de440s` fixture should not be used directly. By marking the test that it is needed in with `@pytest.mark.external_kernel()`, it should automatically be available. See the usage here: https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/dev/imap_processing/tests/spice/test_geometry.py#L36